phpunit/blocks/render-block-rss-test.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Tests for core/rss Gutenberg block.
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ */
+
+/**
+ * Class for testing the core/rss Gutenberg block.
+ *
+ * @group blocks
+ */
+class Tests_Blocks_Render_Rss extends WP_UnitTestCase {
+
+	/**
+	 * Set up before each test.
+	 */
+	public function set_up() {
+		parent::set_up();
+
+		add_filter( 'wp_feed_cache_transient_lifetime', '__return_zero' );
+		add_filter( 'pre_http_request', array( $this, 'mock_http_request' ), 10, 3 );
+	}
+
+	/**
+	 * Mock HTTP request to return test feed data.
+	 *
+	 * @ticket 62400
+	 *
+	 * @param bool|array $response The existing response or false.
+	 * @param array      $args     The request arguments.
+	 * @param string     $url      The request URL.
+	 * @return array The mocked response.
+	 */
+	public function mock_http_request( $response, $args, $url ) {
+		if ( 'https://example.com/testrss.xml' !== $url ) {
+			return $response;
+		}
+
+		return array(
+			'headers'  => array(
+				'content-type' => 'application/rss+xml; charset=UTF-8',
+			),
+			// phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
+			'body'     => file_get_contents( GUTENBERG_DIR_TESTDATA . 'feed/feed-with-gmt-offset.xml' ),
+			'response' => array(
+				'code'    => 200,
+				'message' => 'OK',
+			),
+			'cookies'  => array(),
+			'filename' => null,
+		);
+	}
+
+	/**
+	 * Sets up the "core/rss" block context for testing.
+	 * This is needed to avoid null access in WP_Block_Supports::apply_block_supports().
+	 *
+	 * @ticket 62400
+	 */
+	private function setup_block_context() {
+		$block = array(
+			'blockName' => 'core/rss',
+			'attrs'     => array(),
+		);
+
+		$wp_block_supports = WP_Block_Supports::get_instance();
+		$reflection        = new ReflectionClass( $wp_block_supports );
+		$property          = $reflection->getProperty( 'block_to_render' );
+		$property->setAccessible( true );
+		$property->setValue( $wp_block_supports, $block );
+	}
+
+	/**
+	 * Test that the date in the RSS feed is correctly rendered in the HTML.
+	 *
+	 * @ticket 62400
```

Comment:
Reference to the Gutenberg ticket number rather than the core ticket.

```suggestion
	 * @ticket 66970
```